src/coreclr/nativeaot/Runtime/eventpipe/ep-rt-aot.cpp

```diff
@@ -834,11 +834,23 @@ ep_rt_thread_handle_t ep_rt_aot_thread_get_handle (void)
     return ThreadStore::GetCurrentThreadIfAvailable();
 }
 
+ep_rt_thread_handle_t ep_rt_aot_thread_get_handle_for_streaming (void)
```

Comment:
```suggestion
ep_rt_thread_handle_t ep_rt_aot_setup_thread (void)
```
This is not a simple getter. The name should reflect that.